js/src/run_trees.ts

```diff
@@ -61,6 +61,11 @@ export interface RunTreeConfig {
 }
 
 export interface RunnableConfigLike {
+  /**
+   * Name of the traced run, overriding the name provided by the traceable
+   * function.
+   */
+  name?: string;
```

Comment:
Haha the original intent of this was for the langchain -> langsmith handoff. I'm guessing the scope has crept a bit?